docs/clients/multi_turn_angle_control.rst

```diff
@@ -118,69 +118,74 @@ Message Table
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 6      | angle_Ki                        | get, set, save | float     | V/(rad*s) | Integral gain                                                                                                                                                                                                                                                                                                                                      |
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 7      | angle_Kd                        | get, set, save | float     | V/(rad/s) | Derivative gain                                                                                                                                                                                                                                                                                                                                    |
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 8      | timeout                         | get, set, save | float     | s         | The controller must receive a message within this time otherwise it is set to coast mode                                                                                                                                                                                                                                                           |
```

Comment:
Add period at the end

Reply:
I think we inherited this from whenever the table was written, so the inconsistencies make sense. I'll fix them